paasta_tools/contrib/ide_helper.py

```diff
@@ -99,6 +111,45 @@ def install_vscode_support() -> None:
                     "canary",
                 ],
             },
+            {
```

Comment:
not necessary to do now: given how fast/quickly this is growing it might be nice to move these dicts to separate json files rather than inlining the data here